contracts/FlowToken.cdc

```diff
@@ -83,6 +86,24 @@ pub contract FlowToken: FungibleToken {
                 FlowToken.totalSupply = FlowToken.totalSupply - self.balance
             }
         }
+
+        /// Get all the Metadata Views implemented by FlowToken
+        ///
+        /// @return An array of Types defining the implemented views. This value will be used by
+        ///         developers to know which parameter to pass to the resolveView() method.
+        ///
+        pub fun getViews(): [Type]{
+            return FlowToken.getViews()
+        }
+
+        /// Get a Metadata View from FlowToken
+        ///
+        /// @param view: The Type of the desired view.
+        /// @return A structure representing the requested view.
+        ///
+        pub fun resolveView(_ view: Type): AnyStruct? {
+            return FlowToken.resolveView(view)
+        }
```

Comment:
I feel like `balance` should be somewhere in some view, and we should resolve that view here.

Reply:
`balance` is already accessible directly on the `Vault`, so there is no need to put it into a view, right?

Reply:
yeah you are right